control-plane/subcommand/server-acl-init/command_ent_test.go

```diff
@@ -1031,7 +1031,7 @@ func TestRun_NamespaceEnabled_ValidateLoginToken_PrimaryDatacenter(t *testing.T)
 		{
 			ComponentName: "connect-injector",
 			TokenFlags:    []string{"-connect-inject"},
-			Roles:         []string{resourcePrefix + "-connect-injector-acl-role"},
+			Roles:         []string{resourcePrefix + "-connect-inject-acl-role"},
```

Comment:
I am curious why this changed?

Reply:
Before we used to derive the role name from the service account of the component. With snapshot agent now being co-located with servers, the service account would be the server's service account. I changed it to instead use the component name that we pass to these functions so that both policy and role are derived the same way and so that the role will still clearly say that it's for the snapshot agent. This resulted in ripple effects for other roles which were not named the same as the component.